1_assignment/alphbet_frequency.js

```diff
@@ -0,0 +1,51 @@
+
+string = "u@#d$fkj$#!kds"
+
+function calculateFrequency(string) {
+  // Write your code here
+  var i=0;
+  frequency=new Array(26).fill(0);
+  for (i =0; i< string.length ; i++)
+  {
+  	if (string.charCodeAt(i) >= 97 && string.charCodeAt(i)<=122)
+  	{
+  		frequency[(string.charCodeAt(i)-97)]++;
+  	}
+  }
+  
+  return frequency;
+}
+
+function calculateFrequency2(string) {
```

Comment:
Function names should be descriptive. calculateFrequency2() does not convey any meaning